pydantic_ai_slim/pydantic_ai/models/anthropic.py

```diff
@@ -545,6 +546,9 @@ async def _map_message(self, messages: list[ModelMessage]) -> tuple[str, list[Be
                                         ),
                                     )
                                 )
+                    elif isinstance(response_part, FilePart):  # pragma: no cover
+                        # Images generated by models are not sent back to models that don't themselves generate images.
```

Comment:
```suggestion
                        # Files generated by models are not sent back to models that don't themselves generate files.
```
since it says FilePart it seems this should apply to more than just images in principle?